app/src/fs_utils.c

```diff
@@ -0,0 +1,60 @@
+#include "fs_utils.h"
```

Comment:
Was this ever tested? If not, I would unstage the header and src and implement and test these in a new PR.

Reply:
Everything in `app` was what was there before, the only changes I made were to app_nucleo (later samples/logging).